### ISSUE TITLE
getDelegation error Blockfrost API

### ISSUE THREAD

**Issue by albertorizzi** [spacebudz/lucid#88]
I try to get delegation info with this method:

```
let delegation =  await lucid.wallet.getDelegation();
console.log("delegation", delegation);
```
I obtain `null` information despite I've delegate to a pool.

The problem I think it's in the Blockfrost Api. Infact, I think the address passed is in _raw address format_, but [Bockfrost Api](https://docs.blockfrost.io/#tag/Cardano-Accounts/paths/~1accounts~1{stake_address}/get) would have a address in _stake format_.

Here a screenshot of console error.

![image](https://user-images.githubusercontent.com/48057685/193611248-2d077e5c-4b49-4680-9cd1-cf2d299c0ef4.png)

**Comment by albertorizzi** [comment]
Hi @alessandrokonrad, do you confirm this error?

**Comment by alessandrokonrad** [comment]
Hey @albertorizzi I'm not facing this issue. What network are you on? Also are you sure your stake key is registered and delegated to a pool?

**Comment by albertorizzi** [comment]
I'm on Mainnet and I'm delegate to a pool. This is my [address](https://cardanoscan.io/address/011db7e8e3c128c1a3711c7326b232231c98441149041349ee8e0282ece4ff47a36e9602fdee192181be146d445591a051d56b5615fe3e7d42). Can I do other tests?

**Comment by albertorizzi** [comment]
@alessandrokonrad I think the issue is in this line: https://github.com/spacebudz/lucid/blob/e0bcb052a8d742e41740c3e2ea1da52bca2377ed/src/provider/blockfrost.ts#L128

Blockfrost method require address in stake format, not address passed is in raw address format

**Comment by alessandrokonrad** [comment]
Mhhh I'm not sure why you even get the stake address in hex and not bech32 there. That should not be possible. When I do `lucid.wallet.getDelegation()` it queries from the blockfrost endpoint correctly with the bech32 format.
What method did you use to select a wallet in Lucid? e.g. lucid.selectWalletFromSeed

**Comment by albertorizzi** [comment]
> What method did you use to select a wallet in Lucid? e.g. lucid.selectWalletFromSeed

I do `selectWallet` method. 

`
  const enableWallet = async (walletName) => { 
     if (lucid) { 
       const api = await window.cardano[walletName].enable(); 
       lucid.selectWallet(api); 
     } 
   }
`

Do you think it could be here the problem?

**Comment by alessandrokonrad** [comment]
I see it now. There was indeed a little issue in the selectWallet method when constructing the stake address. Thanks!

**Comment by alessandrokonrad** [comment]
Should be resolved now with Lucid 0.6.7

**Comment by albertorizzi** [comment]
@alessandrokonrad Thank you very much Ales! And thank you for Lucid! Great!
